ci-operator/step-registry/openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-ref.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "openshift-qe/kube-burner-smoke/openshift-qe-kube-burner-smoke-chain.yaml",
+	"path": "openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-ref.yaml",
```

Comment:
@krishvoor Should this be control-plane-smoke-chain or control-plane-smoke-ref?

Reply:
Same as other comment.

Reply:
> @krishvoor Should this be control-plane-smoke-chain or control-plane-smoke-ref?

hmm this was autogenerated, and the checks were clean - lemme see if I have any indicators of ref.

Reply:
> @krishvoor Should this be control-plane-smoke-chain or control-plane-smoke-ref?

The path is pointing to a yaml file reference accessible via `"openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-ref.yaml",` accessed via:

https://github.com/openshift/release/blob/a40070c27776c297a0767a7f3d70682ee7a880fb/ci-operator/step-registry/openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-ref.yaml